src/app/lib/analyticsUtils/chartbeat.js

```diff
@@ -0,0 +1,48 @@
+import Cookie from 'js-cookie';
+import onClient from '../utilities/onClient';
+
+const ID_COOKIE = 'ckns_sylphid';
+
+export const chartbeatUID = 50924;
+export const useCanonical = true;
+
+const buildSectionArr = (service, value, type) => [
+  `${service} - ${value}`,
+  `${service} - ${value} - ${type}`,
+];
+
+const buildServiceType = (service, type) => `${service} - ${type}`;
+
+const capitalize = s => s.charAt(0).toUpperCase() + s.slice(1);
+
+export const getSylphidCookie = () => {
+  if (!onClient()) {
+    return null;
+  }
+
+  return Cookie.get(ID_COOKIE) || null;
```

Comment:
According to docs `Cookie.get` can return undefined, is there a reason we want to convert anything falsy to `null`?

Reply:
Good spot. This tidies the function up nicely